frontend/admin/webpack.base.js

```diff
@@ -13,35 +13,12 @@ module.exports = {
   entry: {
     app: [
       "./src/js/main.tsx",
-      "../common/src/css/hydrogen.css",
+      path.resolve(__dirname, "../common/src/css/hydrogen.css"),
       "../common/src/css/common.css",
       "./src/css/app.css",
     ],
   },
   plugins: [
-
-    // Run Hydrogen on Webpack's compiler hooks
```

Comment:
Whoah, is Hydrogen in the turbo pipeline now?  Will it still work in watch mode?

Reply:
Doh! It was running but only at the start of a watch. I made some changes and it should be running during watch now.

[improve watch commands](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5173/commits/2e9eb52dac3d44deed80555c0ef4614e38a29b9e)